openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KRoom.java

```diff
@@ -237,7 +237,7 @@ public void startSharing(StreamProcessor processor, IClientManager cm, Client c,
 			sd = c.addStream(StreamType.SCREEN, a);
 			sd.setWidth(msg.getInt("width")).setHeight(msg.getInt("height"));
 			cm.update(c);
-			log.debug("User {}: has started sharing", sd.getUid());
+			log.debug("User {}: has started sharing, with stream: {}", sd.getUid(), a);
```

Comment:
I Would change this message to be `Stream {}: sharing has been started, activity: {}`